tests/v4l2decode.cpp

```diff
@@ -330,6 +329,153 @@ static bool displayOneVideoFrameAndroid(int32_t fd, int32_t index)
 
     return true;
 }
+
+#elif __ENABLE_WAYLAND__
```

Comment:
yes, how about merge it firstly? and we take some time to refine the code in the future.